packages/sdk/src/swap/types.ts

```diff
@@ -57,7 +57,8 @@ export interface CommonStatusFields extends ChainsAndAssets {
   depositAddress: string | undefined;
   depositChannelCreatedAt: number | undefined;
   expectedDepositAmount: string | undefined;
-  depositChannelExpiryTime: number | undefined;
+  depositChannelExpiryBlock: bigint;
```

Comment:
this info might be useful for consumers that dont want to rely on the time estimation. also we already expose it from the `requestDepositAddress` method